src/pages/login/index.js

```diff
@@ -3,17 +3,40 @@ import Button from "../../components/button";
 import TextInput from "../../components/form/textInput";
 import useAuth from "../../hooks/useAuth";
 import CredentialsCard from "../../components/credentials";
+import { validateEmail } from "../../utils/validations";
 import "./login.css";
 
 const Login = () => {
   const { onLogin } = useAuth();
   const [formData, setFormData] = useState({ email: "", password: "" });
+  const [errors, setErrors] = useState({ email: "", password: "" });
+
+  const isValidPassword = (password) => {
```

Comment:
why is this not in the utils/validations? i'd imagine it would be re-used in login / registration?

Reply:
There is a function called validatePassword that handles the validation now instead of isValidPassword. 

I Should have removed this from the login component as it is now redundant here.